test/repositories/switch-repository.test.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+import { DynamoDBClient, DynamoDBClientConfig } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient } from '@aws-sdk/lib-dynamodb';
+
+import { SynthSwitchRequestBody } from '../../lib/handlers/synth-switch';
+import { SwitchRepository } from '../../lib/repositories/switch-repository';
+
+const dynamoConfig: DynamoDBClientConfig = {
+  endpoint: 'http://localhost:8000',
+  region: 'local',
+  credentials: {
+    accessKeyId: 'fakeMyKeyId',
+    secretAccessKey: 'fakeSecretAccessKey',
+  },
+};
+
+const SWITCH: SynthSwitchRequestBody = {
+  inputToken: 'USDC',
+  outputToken: 'UNI',
+  inputTokenChainId: 1,
+  outputTokenChainId: 1,
+  amount: '1000000000000000000',
+  type: 'EXACT_INPUT',
+};
+
+const NONEXISTENT_SWITCH: SynthSwitchRequestBody = {
+  inputToken: 'USDC',
+  outputToken: 'UNI',
+  inputTokenChainId: 1,
+  outputTokenChainId: 1,
+  amount: '1000000000000000000',
+  type: 'EXACT_OUTPUT',
+};
+
+const documentClient = DynamoDBDocumentClient.from(new DynamoDBClient(dynamoConfig), {
+  marshallOptions: {
+    convertEmptyValues: true,
+  },
+  unmarshallOptions: {
+    wrapNumbers: true,
+  },
+});
+
+const switchRepository = SwitchRepository.create(documentClient);
+
+describe('put switch tests', () => {
+  it('should put synth switch into db', async () => {
```

Comment:
nit: should we add a unit test for the case when we have multiple trade sizes of the same swap in the DB?